tests/by-util/test_printf.rs

```diff
@@ -293,7 +298,7 @@ fn sub_num_float_e_no_round() {
 #[test]
 fn sub_num_float_round() {
     new_ucmd!()
-        .args(&["two is %f", "1.9999995"])
+        .args(&["two is %f", "1.9999996"])
```

Comment:
That's cheating ;-)

Reply:
I thought it wasn't cheating because my `printf` said:
```
❯ printf "two is %f" 1.9999995
two is 1,999999
```
but I just tried with
```
❯ env printf "two is %f" 1.9999995
two is 2,000000
```
So yeah I should change that back 😄

Reply:
I've ignored this test for now. I did add a test for `0.9999995` as well so that we at least have a test which checks for rounding (and which does work).

Reply:
maybe check for the two potential values ?